apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeValidations.tsx

```diff
@@ -87,47 +86,48 @@ export const AttributeValidations = () => {
           {t("realm-settings:addValidator")}
         </Button>
         <Divider />
-        <TableComposable aria-label="validators-table">
-          <Thead>
-            <Tr>
-              <Th>{t("validatorColNames.colName")}</Th>
-              <Th>{t("validatorColNames.colConfig")}</Th>
-              <Th />
-            </Tr>
-          </Thead>
-          <Tbody>
-            {validators.map((validator) => (
-              <Tr key={validator.key}>
-                <Td dataLabel={t("validatorColNames.colName")}>
-                  {validator.key}
-                </Td>
-                <Td dataLabel={t("validatorColNames.colConfig")}>
-                  {JSON.stringify(validator.value)}
-                </Td>
-                <Td className="kc--attributes-validations--action-cell">
-                  <Button
-                    key="validator"
-                    variant="link"
-                    data-testid="deleteValidator"
-                    onClick={() => {
-                      toggleDeleteDialog();
-                      setValidatorToDelete({
-                        name: validator.key,
-                      });
-                    }}
-                  >
-                    {t("common:delete")}
-                  </Button>
-                </Td>
+        {validators.length !== 0 ? (
+          <TableComposable aria-label="validators-table">
```

Comment:
Same here, not readable for humans.
```suggestion
          <TableComposable>
```